src/utils/FetchConfig.ts

```diff
@@ -95,7 +95,8 @@ export async function FetchConfig (): Promise<any> {
   }
 
   const iaSurveyId: string = response.data.IA_SURVEY_ID
-  sessionStorage.setItem('IA_SURVEY_ID', iaSurveyId)
+  // NB: assign fallback otherwise sessionStorage will store string "undefined"
+  sessionStorage.setItem('IA_SURVEY_ID', iaSurveyId || '')
```

Comment:
`sessionStorage.setItem('IA_SURVEY_ID', null)` will save a string value of "null".

`sessionStorage.setItem('IA_SURVEY_ID', undefined)` will save a string value of "undefined".

So if you want a falsy value in there, you need to set empty string.